### ISSUE TITLE
Presentation Thumbnails

### ISSUE THREAD

**Issue by sebastian-meier** [frntl/frontal#40]
One thing that would be really nice to have a thumbnail view of your presentation (within the note window).

I have done this with node before using phantom. The idea would be to quickly generate those tumbnails when the user opens the presentation.

I would store the thumbnails in a temporary folder within the user/app support/FRONTAL/ folder.

**Comment by fabiantheblind** [comment]
As temp files? We should then take care to empty this folder. Or we trash the users system with thumbs :-)

**Comment by sebastian-meier** [comment]
Yes, indeed. Every time somebody opens a new presentation we clear the
whole folder, i would suggest.

Fabian Morón Zirfas <notifications@github.com> schrieb am Di., 17. Nov.
2015 um 11:39 Uhr:

**Comment by sebastian-meier** [comment]
As a future todo we could of course do this on app close event (if there is
such a thing).

Sebastian Meier <contact@sebastianmeier.eu> schrieb am Di., 17. Nov. 2015
um 11:46 Uhr:

**Comment by fabiantheblind** [comment]
okidoki

**Comment by fabiantheblind** [comment]
What about this https://www.npmjs.com/package/html-to-pdf

**Comment by fabiantheblind** [comment]
https://github.com/jameskyburz/electron-screenshot

**Comment by fabianmoronzirfas** [comment]
https://github.com/oliver-moran/jimp this non native module might be better then the current solution
